src/components/switch/bl-switch.css

```diff
@@ -40,6 +41,32 @@ span {
   width: var(--thumb-width);
 }
 
+label {
+  display: flex;
+  gap: var(--bl-size-2xs);
+  color: var(--bl-color-neutral-darker);
+  font: var(--bl-font-title-3);
+  cursor: pointer;
+  user-select: none;
+  line-height: normal;
+  align-items: var(--bl-radio-align-items, center);
```

Comment:
We shouldn't use another components var.
Since we are not allowing user to change flex direction, might as well ditch it.

Reply:
Yes, you are right, I will pay attention to this in my other contributions as well. Thank you. I changed it as you said.